crates/relayer-types/src/core/ics24_host/identifier.rs

```diff
@@ -18,7 +18,7 @@ use crate::core::ics24_host::error::ValidationError;
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 #[serde(from = "tendermint::chain::Id", into = "tendermint::chain::Id")]
 pub struct ChainId {
-    id: String,
+    pub id: String,
```

Comment:
You can call `as_str()` in `ChainId` instead of accessing the `id` field directly

Reply:
I specifically wanted the name of the chain, i.e. everything before the dash. It looks like the `id` field already includes both the chain name and the version, so I added a `name` method in [9f63875](https://github.com/informalsystems/hermes/pull/3466/commits/9f63875e205e2bddf241cb8c97c90955163bc733)